src/components/Footer.svelte

```diff
@@ -78,66 +78,65 @@
         <span class="sitemap-title">Learn</span>
         <ul class="sitemap-list">
           <li>
-            <a>Blog</a>
-
+            <a href=".">Blog</a>
```

Comment:
`href="#"` seems a better placeholder.

Reply:
href value of `#`/`.` is resolved unexpectedly due to the use of `base` element. Let's use `/` as the placeholder href value for now.